src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1943,6 +1943,32 @@
                 --advertised-public-prefixes 2002:db00::/126
           min_profile: latest
 """
+
+helps['network express-route peering connection'] = """
+    type: group
+    short-summary: Manage ExpressRoute peering connections.
+"""
+
+helps['network express-route peering connection create'] = """
+    type: command
+    short-summary: Create connections between two ExpressRoute circuits.
+    examples:
+        - name: Create connection between two ExpressRoute circuits with AzurePrivatePeering settings.
+          text: |
+            az network express-route peering connection create -g MyResourceGroup --circuit-name \\
+                MyCircuit --peering-name AzurePrivatePeering -n myConnection --peer-circuit \\
+                MyOtherCircuit --address-prefix 104.0.0.0/29
```

Comment:
should we add an example for authorization key if the peer circuit is in different subscription

Reply:
I think it would make more sense to put that in the help text for `authorization-key`.